src/vue/pages/StatisticsSalesHistory.vue

```diff
@@ -0,0 +1,161 @@
+<template>
```

Comment:
make changes how in the previous component

Reply:
used